packages/teams-js/src/internal/responseHandler.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * This class is used for validating and deserializing the response from the host.
+ *
+ * @typeParam ReceivedFromHost The type of the response received from the host
+ * @typeParam DeserializedFromHost The type of the response after deserialization
+ */
+export abstract class ResponseHandler<ReceivedFromHost, DeserializedFromHost> {
```

Comment:
>ReceivedFromHost, DeserializedFromHost [](http://example.com/codeflow?start=38&length=38)

I left this in the communication.ts file but I now realize I should probably have put it here -- but I don't want to delete my original comment since Jay responded to it.

Consider naming these types something like `SerializedReturnValueFromHost` and `DeserializedReturnValueFromHost`